fury/tests/test_actors.py

```diff
@@ -1819,3 +1820,84 @@ def test_actors_primitives_count():
         primitives_count = test_case[2]
         act = act_func(**args)
         npt.assert_equal(primitives_count_from_actor(act), primitives_count)
+
+def test_kde_actor(interactive = False):
+    width, height = (400, 400)
```

Comment:
maybe you could use `size = ...` as you are not using the width and height individually